<launch>  

  <arg name="urdf_filename" default="$(find galileo_go1_description)/urdf/go1.urdf" />  
  <arg name="solver_id" default="galileo_go1" />
  <arg name="problem_parameters_location" default="$(find galileo_go1_description)/Parameters/problem_parameters.txt" />
  <arg name="robot_body_frame_name" default="base" />

  <param name="galileo_ros/urdf_filename" value="$(arg urdf_filename)"/>
  <param name="robot_description" command = "$(find xacro)/xacro --inorder $(find galileo_go1_description)/urdf/go1.urdf" />
  <param name="galileo_ros/solver_id" value="$(arg solver_id)"/>
  <param name="galileo_ros/problem_parameters_location" value="$(arg problem_parameters_location)"/>
  <param name="galileo_ros/robot_body_frame_name" value="$(arg robot_body_frame_name)"/>  

  <node pkg="robot_state_publisher" type="robot_state_publisher" name="robot_state_publisher">
    <!-- <param name="ignore_timestamp" type="bool" value="true"/> -->
  </node>

  <node name="rviz" pkg="rviz" type="rviz" output="screen" args="-d $(find galileo_ros)/rviz/go1.rviz" />


  <node name="galileo_ros_legged_rviz_node" type="galileo_ros_legged_rviz_node" pkg="galileo_ros" output="screen">
  </node>


  <node name="galileo_ros_legged_verbose_rviz" type="galileo_ros_legged_verbose_rviz" pkg="galileo_ros" output="screen">  
  </node>

  
</launch>